builtin/logical/pki/integration_test.go

```diff
@@ -381,12 +386,15 @@ func TestIntegration_AutoIssuer(t *testing.T) {
 	requireSuccessNonNilResponse(t, resp, err)
 	require.Equal(t, issuerIdOne, resp.Data["default"])
 
+	schema.ValidateResponse(t, schema.FindResponseSchema(t, paths, 0, logical.ReadOperation), resp, true)
```

Comment:
If it's not too much work, could you please switch to `schema.GetResponseSchema(t,...)` (with an optional `b.Route(...)`) for these. A better example can be found in https://github.com/hashicorp/vault/pull/18865.

Reply:
yeah. Ezpz. Finding the tests was the most difficult part, so I can move to this implementation.

Reply:
okay. I think I got to them all